src/main/java/com/it4045/pokemonviewer/enterprise/service/IPokemonService.java

```diff
@@ -5,7 +5,16 @@
 import java.util.List;
 
 public interface IPokemonService {
-
+    /**
+     * Fetches pokemon by name.
+     * @param pokemonName
+     * @return Pokemon
+     */
```

Comment:
good practice to have JavaDoc's for your methods.

Reply:
JavaDoc is the most valuable when it adds description beyond what can be read from code.
Use the @param tag to describe the implicit parameters
Use the @returns tag to describe what the method returns.